external/mmdetection/detection_tasks/apis/detection/inference_task.py

```diff
@@ -169,10 +169,10 @@ def _create_model(config: Config, from_scratch: bool = False):
             model = build_detector(model_cfg)
         return model
 
-
-    def _add_predictions_to_dataset(self, prediction_results, dataset, confidence_threshold=0.0):
+    def _add_predictions_to_dataset(self, prediction_results, dataset, confidence_threshold=0.0,
```

Comment:
this method does not return anything, probably `return_saliency_map` is not the best name